src/pages/profiles/EditProfileForm.tsx

```diff
@@ -170,7 +170,11 @@ const EditProfileForm: FC<Props> = ({ profile, featuresProfiles }) => {
     <div className="edit-profile">
       {!featuresProfiles && (
         <Notification severity="caution" title="Inherited profile">
-          Modifications are only available in the default project.
+          Modifications are only available in the{" "}
+          <Link to={`/ui/project/default/profiles/detail/${profile.name}`}>
```

Comment:
Should this redirect to the configuration tab?
```suggestion
          <Link to={`/ui/project/default/profiles/detail/${profile.name}/configuration`}>
```